providers/openebs/installers/storageclass/0.6/ansible/storageclass_setup.yaml

```diff
@@ -0,0 +1,54 @@
+# TODO 
+# Apply storage pool and storage class
+
+- hosts: localhost
+  connection: local
+
+  vars_files:
+  - vars.yaml
+
+  tasks:
+    - block:
+          - template:
+              src: sp.j2
+              dest: "{{ storage_pool }}"
+
+          - template:
+              src: sc.j2
+              dest: "{{ storage_class }}"
+
+          - name: Applying storage pool
+            shell: "{{ kubeapply }} apply -f {{ storage_pool }}" 
+            args:
+              executable: /bin/bash
+            ignore_errors: True

```

Comment:
@ashishranjan738 , any particular reason for adding the ignore_errors (are you possibly factoring in existing pools/sc with the same names? )

Reply:
Got it. We should be OK for actual deploy failures as the job will fail due to validations for the resources in subsequent tasks. However, thinking about this case: 

- It is possible that the CI has passed custom values for the params via ENV (diskpaths, replica counts etc.,), which may/may-not be present in the existing SP/SC. That way, the application deployment job will be affected. 

What do you think about handling it by removing the existing class and launching newly. Even if there are existing application PVCs using the same storage class, it should be OK as the pods will be already deployed.

Reply:
ok so let's stop our job as a failure when we find any previous sc or sp artifacts.